platformio.ini

```diff
@@ -21,35 +21,44 @@
 env_default = emonesp
 data_dir = src/data
 
+[common]
+version = -DBUILD_TAG=2.0.0
+#MQTT: https://github.com/knolleary/pubsubclient
+lib_deps = PubSubClient@2.6
```

Comment:
Personally I have a pet hate of magic numbers so I always try to remove them ;-)